Sources/TuistGenerator/Generator/Generator.swift

```diff
@@ -57,7 +59,7 @@ public class Generator: Generating {
     public convenience init(defaultSettingsProvider: DefaultSettingsProviding = DefaultSettingsProvider(),
                             modelLoader: GeneratorModelLoading) {
         let graphLinter = GraphLinter()
-        let graphLoader = GraphLoader(linter: graphLinter, modelLoader: modelLoader)
```

Comment:
is there any specific reason why we have linter taken outside `graphLoader`?

Reply:
To not consider linting as part of loading. That way the responsibility of the loader is just loading the models, nothing else.